utils/firestore.js

```diff
@@ -2201,72 +2201,73 @@ const setPackageReceiptUSPS = async (data) => {
         
-        // Updated BagConceptIDs for new 40 bag capacity. Retained these because they were previously included.
-        const additionalBagItems = ["255283733", "842312685", "234868461", "522094118"];
-
-        const allBagsToCheck = [...bagConceptIDs, ...additionalBagItems];
-        if (Object.keys(snapshot.docs.length) !== 0) {
-             snapshot.docs.map(doc => { 
-                const collectionIdKeys = doc.data(); // grab all the collection ids
-                allBagsToCheck.forEach(async (bag) => {
-                    if (bag in collectionIdKeys){
-                        if (collectionIdKeys[bag]['787237543'] !== undefined || collectionIdKeys[bag]['223999569'] !== undefined || collectionIdKeys[bag]['522094118'] !== undefined) {
-                            if (collectionIdKeys[bag]['787237543'] !== `` && collectionIdKeys[bag]['223999569'] === `` && collectionIdKeys[bag]['522094118'] === ``) {
-                                collectionIdHolder[bag] = collectionIdKeys[bag]['787237543'].split(' ')[0]
-                            }
-                            if (collectionIdKeys[bag]['223999569'] !== `` && collectionIdKeys[bag]['787237543'] === `` && collectionIdKeys[bag]['522094118'] === ``) {
-                                collectionIdHolder[bag] = collectionIdKeys[bag]['223999569'].split(' ')[0]
-                            }
-                            if (collectionIdKeys[bag]['522094118'] !== `` && collectionIdKeys[bag]['223999569'] === `` && collectionIdKeys[bag]['787237543'] === `` ) {
-                                collectionIdHolder[bag] = collectionIdKeys[bag]['522094118'].split(' ')[0]
-                            }
+        for (const doc of snapshot.docs) {
+            const collectionIdKeys = doc.data(); // grab all the collection ids
+            for (const bag of bagConceptIDs) {
+                if (bag in collectionIdKeys){
+                    if (collectionIdKeys[bag]['787237543'] !== undefined || collectionIdKeys[bag]['223999569'] !== undefined || collectionIdKeys[bag]['522094118'] !== undefined) {
+                        if (collectionIdKeys[bag]['787237543'] !== `` && collectionIdKeys[bag]['223999569'] === `` && collectionIdKeys[bag]['522094118'] === ``) {
+                            collectionIdHolder[bag] = collectionIdKeys[bag]['787237543'].split(' ')[0]
+                        }
+                        if (collectionIdKeys[bag]['223999569'] !== `` && collectionIdKeys[bag]['787237543'] === `` && collectionIdKeys[bag]['522094118'] === ``) {
+                            collectionIdHolder[bag] = collectionIdKeys[bag]['223999569'].split(' ')[0]
+                        }
+                        if (collectionIdKeys[bag]['522094118'] !== `` && collectionIdKeys[bag]['223999569'] === `` && collectionIdKeys[bag]['787237543'] === `` ) {
+                            collectionIdHolder[bag] = collectionIdKeys[bag]['522094118'].split(' ')[0]
```

Comment:
It absolutely can be - this was implemented over a year ago, and I didn't want to touch it without more time to test & analyze since it's going in the current push and it was found last-minute. I would prefer to flag it for an update but not push this type of change last minute if that works for you.